lint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: ./lint.sh path/to/html/source"
+  exit 1
+fi
+
+# show potential problems
+# note - would be nice if the ones with \s+ patterns actually cross lines, but, they don't...
+MATCHES=$(grep -ni 'xxx' $1 | perl -lpe 'print "\nPossible incomplete sections:" if $. == 1' \
+  & grep -niE '( (code|span|var)(>| data-x=)|[^<;]/(code|span|var)>)' $1 | perl -lpe 'print "\nPossible copypasta:" if $. == 1' \
```

Comment:
Won't `&` cause the greps to run in parallel and change the order in a racy way? A deliberate optimization? Just `;` should make it serial.